.ci/pytorch/windows/cuda128.ps1

```diff
@@ -0,0 +1,54 @@
+# Import common functions
+. "$PSScriptRoot\common.ps1"
```

Comment:
can we try to rewrite it so that we don't need to add cuda12X.ps1 every time so that we can specify CUDA version as a param and reuse this script across all cuda verisons ?